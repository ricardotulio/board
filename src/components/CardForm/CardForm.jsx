import React from 'react'
import styles from './styles.module.css'

const CardForm = () => (
  <form>
    <label className={ styles.input }>
      Title: 
      <input name="title" type="text" />
    </label>
    <label className={ styles.input }>
      Description:
      <textarea name="description" />
    </label>
    <input type="submit" value="Submit" />
  </form>
)

export default CardForm
