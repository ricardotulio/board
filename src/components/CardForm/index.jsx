export { default } from './CardForm.jsx'
