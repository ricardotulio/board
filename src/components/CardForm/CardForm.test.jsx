import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import CardForm from './CardForm'

it('can render card form', () => {
  const { container } = render(<CardForm />)
  expect(container.firstChild).toMatchSnapshot()
})
