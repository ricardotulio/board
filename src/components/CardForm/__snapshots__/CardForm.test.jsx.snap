// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render card form 1`] = `
<form>
  <label
    class="input"
  >
    Title:
    <input
      name="title"
      type="text"
    />
  </label>
  <label
    class="input"
  >
    Description:
    <textarea
      name="description"
    />
  </label>
  <input
    type="submit"
    value="Submit"
  />
</form>
`;
